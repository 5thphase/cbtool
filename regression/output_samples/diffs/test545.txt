|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1412725095 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1412725113 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1412725153 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|ai_XX |N_x_foo->M_x_bar |A_x_foo->B_x_bar |xxx.xxx.xxx.xxx |1412725170 |none |xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
|Redis Version |2.8.4 
