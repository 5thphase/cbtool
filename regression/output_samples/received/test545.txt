Background operation execution success. Operation "vm-capture" will be processed asynchronously, through the command ""/home/msilva/cloudbench/lib/operations//../../cbact" --procid=TEST_msilva --osp=port:6379,experiment_inst:TEST_msilva,host:9.2.208.21,pid:TEST_msilva,dbid:10,timout:70,protocol:TCP,kind:redis,username:msilva --msp=port:27017,password:cbpwdZ9,timeout:240,host:9.2.208.21,pid:TEST_msilva,database:metrics,protocol:TCP,kind:mongodb,username:msilva --oop=TESTCLOUD,vm_41,none --operation=vm-capture --cn=TESTCLOUD --uuid=72A4BD66-7796-5D67-A290-F4D4DE4E197C --daemon". The process id is 18298.

Going to unconditionally wait for 25s (25 seconds). The command line interface will be blocked during the waiting.
Waited 10 seconds... (40.0%)
Waited 20 seconds... (80.0%)
Waited 25 seconds... (100.0%)
Waited for 25 seconds.
The following ATTACHED AIs are in this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_1      |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |208.223.47.70   |1471276440     |none                                    |9E2ECB82-29AC-5EBB-B97B-C1A8B915BF16
|ai_3      |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |36.24.196.145   |1471276459     |none                                    |DB8C1F54-0075-57B4-BF1C-8C70FDCD8370
|ai_4      |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |35.65.96.198    |1471276501     |none                                    |A19B694D-76F6-553A-9D38-5AF64B0F2B1C
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |230.43.155.54   |1471276517     |none                                    |F52B9735-2EB0-5198-A35F-8726AE289A8C

The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool      |netname    |ai      |aidrs      |uuid
|vm_26         |client_daytrader    |micro32     |208.223.47.70   |simhosta13                 |LG            |private    |ai_1    |none       |ABA4F416-F6B2-55C4-98AF-CD7B22D1ABAE
|vm_28         |db2                 |platinum64  |250.240.235.41  |simhostd9                  |SUT           |private    |ai_1    |none       |61DF07A0-0F9A-5260-8963-74B6702D493E
|vm_34         |was                 |iron32      |161.243.185.92  |simhostb13                 |SUT           |private    |ai_3    |none       |F3BCAC80-3A73-599F-9630-2BE194EB80BB
|vm_33         |client_daytrader    |micro32     |36.24.196.145   |simhostc5                  |LG            |private    |ai_3    |none       |AF8904E1-EAF5-57C6-B10E-1AE4950FDF38
|vm_35         |db2                 |platinum64  |228.90.183.160  |simhostd10                 |SUT           |private    |ai_3    |none       |E1D4ADDE-6D4A-579D-A395-9F3281E0C84A
|vm_36         |client_daytrader    |micro32     |35.65.96.198    |simhosta6                  |LG            |private    |ai_4    |none       |BB8F5420-B3CE-5CBD-B874-9BDA53F3F4D0
|vm_37         |db2                 |platinum64  |163.25.179.139  |simhostb3                  |SUT           |private    |ai_4    |none       |A9FFC26D-4C03-534B-9109-5E892126E6CA
|vm_38         |was                 |iron32      |206.184.149.125 |simhostb9                  |SUT           |private    |ai_4    |none       |19E0A141-99A2-553C-975A-E965B8FAEB6F
|vm_39         |netclient           |platinum64  |230.43.155.54   |simhosta7                  |SUT           |private    |ai_5    |none       |B443C23B-E5F8-5907-8AC1-0E3EE5174AC3
|vm_40         |netserver           |iron32      |139.211.82.124  |simhostc1                  |SUT           |private    |ai_5    |none       |1131F667-D77D-5085-95FB-DF9D225064B9
|vm_43         |was                 |iron32      |90.236.172.215  |simhostb13                 |SUT           |private    |ai_1    |none       |C3C1B636-8E62-5143-A2F7-E19E9ABC3A0B

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |11.96M                        
|Redis Version                                        |3.0.6                         
|Uptime (in seconds)                                  |8705614                       
|Total Connections Received                           |377466                        
|Total Commands Processed                             |1223514271                    
|Number of Keys                                       |1065                          
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|Collections                                          |15                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |60                            
|VMs                                                  |11                            
|AIs                                                  |4                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ISSUED                                          |16                            
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ISSUED                                         |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |11                            
|VMs ISSUED                                           |46                            
|VMs ARRIVED                                          |43                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |32                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|VMs SLA PROVISIONING VIOLATED                        |0                             
|VMs SLA RUNTIME VIOLATED                             |0                             
|VMs APPLICATION ERRORS                               |6                             
|AI RESERVATIONS                                      |4                             
|AIs ISSUED                                           |7                             
|AIs ARRIVED                                          |7                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |3                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ISSUED                                        |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ISSUED                                        |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ISSUED                                         |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |296                           

 ---------------- VMC-WIDE COUNTERS ----------------
|9E74D721-5227-5C9B-A7BD-BE5B4A43D6AA (simzone_d) VM RESERVATIONS|3                             
|3B6C80F6-6318-5721-A484-E3DA77718B57 (simzone_c) VM RESERVATIONS|4                             
|803C5D90-3C59-5871-9ACF-5683E3FB4506 (simzone_a) VM RESERVATIONS|3                             
|270A5605-A7C0-5F5E-B66F-19B77C6645BE (simzone_b) VM RESERVATIONS|1                             

