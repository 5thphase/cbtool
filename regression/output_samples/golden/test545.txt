Background operation execution success. Operation "vm-capture" will be processed asynchronously, through the command "/home/msilva/cloudbench/lib/operations//../../cbact --procid=TEST_msilva --osp=port:6379,experiment_inst:TEST_msilva,host:172.16.0.4,pid:TEST_msilva,dbid:10,timout:70,protocol:TCP,kind:redis,username:msilva --msp=port:27017,password:cbpwdZ9,timeout:240,host:172.16.0.4,pid:TEST_msilva,database:metrics,protocol:TCP,kind:mongodb,username:msilva --oop=TESTCLOUD,vm_41,none --operation=vm-capture --cn=TESTCLOUD --uuid=81FD7081-99B7-5C55-A256-E1C308204E18 --daemon". The process id is 7890.

Going to unconditionally wait for 25s (25 seconds). The command line interface will be blocked during the waiting.
Waited 10 seconds... (40.0%)
Waited 20 seconds... (80.0%)
Waited 25 seconds... (100.0%)
Waited for 25 seconds.
The following ATTACHED AIs are in this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_1      |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |88.229.151.201  |1454604918     |none                                    |CA23D099-7677-5EFA-9919-B5E11033D1B0
|ai_3      |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |109.223.73.173  |1454604937     |none                                    |C211AAA2-A63B-5F87-9901-216AE5372952
|ai_4      |ibm_daytrader  |1_x_client_daytrader->1_x_was->1_x_db2            |28.7.122.40     |1454604982     |none                                    |18F6CE19-98A6-5FBF-8805-D4A23EDDB671
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |208.192.169.1   |1454604999     |none                                    |1088C26B-B699-5F92-86E0-4EABB7C21A2A

The following ATTACHED VMs are in this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool      |netname    |ai      |aidrs      |uuid
|vm_26         |client_daytrader    |micro32     |88.229.151.201  |simhostc4                  |LG            |private    |ai_1    |none       |C28CDCE8-E905-51B8-A380-FBD2DF29299F
|vm_28         |db2                 |platinum64  |114.30.204.122  |simhosta9                  |SUT           |private    |ai_1    |none       |4C6E8FC8-926F-5ABC-8D63-00C5C1EEC008
|vm_33         |client_daytrader    |micro32     |109.223.73.173  |simhostc9                  |LG            |private    |ai_3    |none       |498CA0EE-7E3D-5BA5-85FC-CEDE48215D55
|vm_34         |db2                 |platinum64  |207.241.137.254 |simhostc3                  |SUT           |private    |ai_3    |none       |489BBB5F-4261-5AC9-A003-1D1305F8F9C0
|vm_35         |was                 |iron32      |93.76.208.211   |simhostb2                  |SUT           |private    |ai_3    |none       |82EECCF2-B3ED-582F-9B46-A832FEE6083D
|vm_36         |client_daytrader    |micro32     |28.7.122.40     |simhostc6                  |LG            |private    |ai_4    |none       |648D9305-3E12-5731-82EB-BAA261B77E89
|vm_37         |db2                 |platinum64  |188.250.53.209  |simhostc4                  |SUT           |private    |ai_4    |none       |30A19C15-DE85-5216-8999-E0572AC219B5
|vm_38         |was                 |iron32      |80.57.170.8     |simhostd8                  |SUT           |private    |ai_4    |none       |693B746C-2876-5DF6-B864-1179CD1CD6B0
|vm_39         |netclient           |platinum64  |208.192.169.1   |simhostd4                  |SUT           |private    |ai_5    |none       |2FEE1078-D84E-5546-BD7B-05B00EC06385
|vm_40         |netserver           |iron32      |140.232.249.59  |simhosta8                  |SUT           |private    |ai_5    |none       |D5A26AD2-D940-5861-B307-7C45CA46022A
|vm_43         |was                 |iron32      |8.204.10.233    |simhosta8                  |SUT           |private    |ai_1    |none       |4FC6DEE0-FCC1-57C3-AD85-09058F8E577D

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |19.21M                        
|Redis Version                                        |2.8.6                         
|Uptime (in seconds)                                  |12857201                      
|Total Connections Received                           |548711                        
|Total Commands Processed                             |42767221                      
|Number of Keys                                       |975                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|Collections                                          |28                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |60                            
|VMs                                                  |11                            
|AIs                                                  |4                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ISSUED                                          |16                            
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ISSUED                                         |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |11                            
|VMs ISSUED                                           |46                            
|VMs ARRIVED                                          |43                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |32                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|VMs SLA PROVISIONING VIOLATED                        |0                             
|VMs SLA RUNTIME VIOLATED                             |0                             
|VMs APPLICATION ERRORS                               |7                             
|AI RESERVATIONS                                      |4                             
|AIs ISSUED                                           |7                             
|AIs ARRIVED                                          |7                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |3                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ISSUED                                        |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ISSUED                                        |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ISSUED                                         |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |296                           

 ---------------- VMC-WIDE COUNTERS ----------------
|9ADC8244-8C1F-5728-8581-9D8825A87B6E (simzone_a) VM RESERVATIONS|3                             
|662A8BAF-6270-52A0-B49B-7E1326CCC8B6 (simzone_c) VM RESERVATIONS|1                             
|8E167E6D-B4E1-531D-A90A-A469EF1067C6 (simzone_b) VM RESERVATIONS|4                             
|B37F024B-5D05-5130-8E9E-0804A1122684 (simzone_d) VM RESERVATIONS|3                             

